objcgen/processedtypes.cs

```diff
@@ -189,17 +242,20 @@ public override void ComputeSignatures (Processor p)
 	public class ProcessedProperty: ProcessedMemberBase {
 		public PropertyInfo Property { get; private set; }
 
-		public ProcessedProperty (PropertyInfo property)
+		public ProcessedProperty (PropertyInfo property, Processor processor) : base (processor)
 		{
 			Property = property;
 		}
 
-		public override void ComputeSignatures (Processor p)
+		public override void ComputeSignatures ()
 		{
 			throw new NotImplementedException ();
 		}
 
 		public override string ToString () => Property.ToString ();
+
+		public ProcessedMethod GetMethod => new ProcessedMethod (Property.GetGetMethod (), Processor);
+		public ProcessedMethod SetMethod => new ProcessedMethod (Property.GetSetMethod (), Processor);
```

Comment:
I don't like `=> new` ;-)

This will create a **new** `ProcessedMethod` instance each time it's called and it does not seems efficient (but maybe it's premature optimization). However I can see other issues if we don't deal with equality/gethashcode correctly and more callers create them.